### ISSUE TITLE
NamuFix 이미지 업로드 오류

### ISSUE THREAD

**Issue by jeuse30** [LiteHell/NamuFix#42]
<!DOCTYPE html>
<html>
<head>
<title>파일 올리기 - 나무위키</title>
<meta charset="utf-8">
<meta http-equiv="x-ua-compatible" content="ie=edge">
<meta http-equiv="x-pjax-version" content="42fb3778076e6a647a18895e1c0c1be2">
<meta name="generator" content="the seed">
<meta name="application-name" content="나무위키">
<meta name="mobile-web-app-capable" content="yes">
<meta name="msapplication-tooltip" content="나무위키">
<meta name="msapplication-starturl" content="/w/%EB%82%98%EB%AC%B4%EC%9C%84%ED%82%A4:%EB%8C%80%EB%AC%B8">
<link rel="canonical" href="https://namu.wiki/Upload">
<link rel="search" type="application/opensearchdescription+xml" title="나무위키" href="/opensearch.xml">
<meta name="theme-color" content="#008275"><meta name="viewport" content="user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0, width=device-width">
<link rel="shortcut icon" href="/favicon.ico">
<link rel="icon" sizes="128x128" href="/img/icon.png">
<link rel="apple-touch-icon" href="/img/apple_icon.png">
<meta name="googlebot" content="noarchive">
		

<link rel="stylesheet" href="/css/diffview.css?05e3f5dd">
<link rel="stylesheet" href="/css/katex.min.css?5f0dff2e">
<link rel="stylesheet" href="/css/wiki.css?18f1cd4c">
<link rel="stylesheet" href="/skins/senkawa/css/lib/jquery-ui.min.css?840cca94">
<link rel="stylesheet" href="/skins/senkawa/css/lib/bootstrap.min.css?f040bd43">
<link rel="stylesheet" href="/skins/senkawa/css/lib/ionicons.min.css?2d4234e8">
<link rel="stylesheet" href="/skins/senkawa/css/lib/nprogress.css?9dc52b7d">
<link rel="stylesheet" href="/skins/senkawa/css/bootstrap-fix.css?b43106c0">
<link rel="stylesheet" href="/skins/senkawa/css/layout.css?5fcbe609">
<link rel="stylesheet" href="/skins/senkawa/css/wiki.css?2421e27c">
<link rel="stylesheet" href="/skins/senkawa/css/discuss.css?7fd2aa53">
<link rel="stylesheet" href="/skins/senkawa/css/dark.css?b065ec70">

<script type="text/javascript" src="/js/jquery-2.1.4.min.js?f9c7afd0"></script>

<script type="text/javascript" src="/js/dateformatter.js?508d6dd4"></script>
<script type="text/javascript" src="/js/intersection-observer.js?b462101e"></script>
<script type="text/javascript" src="/js/theseed.js?2389f163"></script>
<script type="text/javascript" src="/skins/senkawa/js/lib/jquery.pjax.js?700f7656"></script>
<script type="text/javascript" src="/skins/senkawa/js/lib/jquery.lazyload.min.js?5c01d7af"></script>
<script type="text/javascript" src="/skins/senkawa/js/lib/jquery-ui.min.js?557e2375"></script>
<script type="text/javascript" src="/skins/senkawa/js/lib/tether.min.js?e21e4520"></script>
<script type="text/javascript" src="/skins/senkawa/js/lib/bootstrap.min.js?a2f0eda8"></script>
<script type="text/javascript" src="/skins/senkawa/js/lib/nprogress.js?5fc208df"></script>
<script type="text/javascript" src="/skins/senkawa/js/namu.js?beddd0b2"></script>
<script src="https://www.google.com/recaptcha/api.js?render=explicit&onload=recaptchaOnLoad" async defer></script>
</head>
<body class="senkawa">

	<script type="text/javascript" src="/skins/senkawa/js/layout.js?a0f91f1a"></script>
	<div class="navbar-wrapper">
		<nav class="navbar navbar-dark bg-inverse navbar-static-top">
			<a class="navbar-brand wiki-logo" href="/w/%EB%82%98%EB%AC%B4%EC%9C%84%ED%82%A4:%EB%8C%80%EB%AC%B8"></a>
			<ul class="nav navbar-nav">
				<li class="nav-item">
					<a class="nav-link" href="/RecentChanges" title="최근 변경">
						<span class="icon ion-compass"></span>
						<span class="icon-title">최근 변경</span>
					</a>
				</li>
				<li class="nav-item">
					<a class="nav-link" href="/RecentDiscuss" title="최근 토론">
						<span class="icon ion-android-textsms"></span>
						<span class="icon-title">최근 토론</span>
					</a>
				</li>
				<li class="nav-item dropdown">
					<a class="nav-link dropdown-toggle" href="#" title="특수 기능" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">
						<span class="icon ion-ios-gear"></span>
						<span class="icon-title">특수 기능</span>

						<span class="caret"></span>
					</a>

					<div class="dropdown-menu right">
						<a class="dropdown-item" href="https://board.namu.wiki/" data-npjax="true">
							<span class="icon ion-clipboard"></span>
							<span class="icon-title">게시판</span>
						</a>

						<div class="dropdown-divider"></div>

						<a class="dropdown-item" href="/NeededPages">
							<span class="icon ion-coffee"></span>
							<span class="icon-title">작성이 필요한 문서</span>
						</a>
						<a class="dropdown-item" href="/OrphanedPages">
							<span class="icon ion-sad-outline"></span>
							<span class="icon-title">고립된 문서</span>
						</a>
						<a class="dropdown-item" href="/UncategorizedPages">
							<span class="icon ion-help-circled"></span>
							<span class="icon-title">분류가 되지 않은 문서</span>
						</a>
						<a class="dropdown-item" href="/OldPages">
							<span class="icon ion-pause"></span>
							<span class="icon-title">편집된 지 오래된 문서</span>
						</a>
						<a class="dropdown-item" href="/ShortestPages">
							<span class="icon ion-heart-broken"></span>
							<span class="icon-title">내용이 짧은 문서</span>
						</a>
						<a class="dropdown-item" href="/LongestPages">
							<span class="icon ion-star"></span>
							<span class="icon-title">내용이 긴 문서</span>
						</a>
						<a class="dropdown-item" href="/BlockHistory">
							<span class="icon ion-eye-disabled"></span>
							<span class="icon-title">차단 내역</span>
						</a>
						<a class="dropdown-item" href="/RandomPage">
							<span class="icon ion-shuffle"></span>
							<span class="icon-title">RandomPage</span>
						</a>
						<a class="dropdown-item" href="/Upload">
							<span class="icon ion-android-upload"></span>
							<span class="icon-title">파일 올리기</span>
						</a>
						<a class="dropdown-item" href="/License">
							<span class="icon ion-pricetag"></span>
							<span class="icon-title">라이선스</span>
						</a>

					</div>
				</li>

				<li class="nav-item">
					<a class="nav-link" href="//namu.live/" title="나무라이브">
						<span class="icon ion-android-list"></span>
						<span class="icon-title">라이브</span>
					</a>
				</li>
			</ul>

			

			<ul class="nav navbar-nav pull-right">
				<li class="nav-item dropdown user-menu-parent">
					<a class="nav-link dropdown-toggle user-menu" href="#" title="사용자:Raeah" data-toggle="dropdown" aria-haspopup="true">
					
						<img class="user-img" src="//secure.gravatar.com/avatar/136b313fc09b5de6592f29d7bc9384c1?d=retro" />
					
					</a>

					<div class="dropdown-menu user-dropdown right">
					
						
							
						
						<div class="dropdown-item user-info">
							<div class="user-info">
								<div>Raeah</div>
								<div>Member</div>
							</div>
						</div>
					
						<div class="dropdown-divider"></div>
						<a id="settingsLink" class="dropdown-item" href="/settings" style="display: none;">설정</a>
						<a id="darkToggleLink" class="dropdown-item" href="#">어두운 화면으로</a>
						<div class="dropdown-divider"></div>
					
						<a class="dropdown-item" href="/mypage">내 정보</a>
						<a class="dropdown-item" href="/w/%EC%82%AC%EC%9A%A9%EC%9E%90:Raeah">내 사용자 문서</a>
						<div class="dropdown-divider"></div>
						<a class="dropdown-item" href="/contribution/author/Raeah/document">내 문서 기여 목록</a>
						<a class="dropdown-item" href="/contribution/author/Raeah/discuss">내 토론 기여 목록</a>
					
						<div class="dropdown-divider"></div>
					
						<a class="dropdown-item" href="/logout?redirect=%2FUpload" data-npjax="true">로그아웃</a>
					
					</div>
				</li>
			</ul>

			<form class="form-inline navbar-form search-box-parent">
				<div class="input-group search-box">
					<span class="input-group-btn left-search-btns">
						<a class="btn btn-secondary" href="/random" role="button">
							<span class="icon ion-shuffle"></span>
						</a>
					</span>
					<input type="text" id="searchInput" class="form-control" placeholder="Search" tabindex="1">
					<span class="input-group-btn right-search-btns">
						<button id="searchBtn" class="btn btn-secondary" type="button">
							<span class="icon ion-search"></span>
						</button>
						<button id="goBtn" class="btn btn-secondary" type="button">
							<span class="icon ion-arrow-right-c"></span>
						</button>
					</span>
				</div>
			</form>
		</nav>
	</div>

	<div class="content-wrapper">
		<aside class="sidebar has-site-notice">
			<div class="card recent-card">
				<h5 class="card-title">최근 변경</h5>
				<div class="link-table" id="recentChangeTable">
					<a><span>00:00</span>갱신중...</a>
				</div>
				<a class="more-link" href="/RecentChanges">[더 보기]</a>
			</div>

			<div class="card live-list-card" data-slug="headline" data-count="5">
				<h5 class="card-title">라이브 헤드라인 최신글</h5>
				<div class="link-table">
					<a><span>00:00</span>갱신중...</a>
				</div>
				<a class="more-link" href="//namu.live/b/headline">[더 보기]</a>
			</div>

			
		</aside>

		<div class="live-topbar-area">&nbsp;</div>
		<article class="container-fluid wiki-article">

			

			
			
				
			
			
				<h1 class="title">
				
					파일 올리기
				
				</h1>
			
			
			

	
	<div class="alert alert-danger alert-dismissible fade in" role="alert">
		<button type="button" class="close" data-dismiss="alert" aria-label="Close">
			<span aria-hidden="true">&times;</span>
			<span class="sr-only">Close</span>
		</button>

		<strong>[오류!]</strong> 이 메시지를 보고 계신다는 것은, 로그인하지 않은 상태에서 문서를 편집하는 중 CAPTCHA를 체크하지 않은 경우입니다. 편집 창으로 되돌아가서, 편집 버튼과 미리보기 버튼 밑에 있는 &#39;로봇이 아닙니다&#39;를 체크하는 것으로 CAPTCHA를 완성하면 정상적으로 편집하실 수 있으며, 동일한 아이피 하에서는 한 번만 체크를 하면 됩니다
	</div>
	

	<form method="post" id="uploadForm" enctype="multipart/form-data" action="/Upload">
		<input type="hidden" name="baserev" value="0" />
		<input type="file" id="fileInput" name="file" hidden />
		<input type="hidden" name="identifier" value="m:Raeah" />

		<div class="row">
			<div class="col-xs-12 col-md-7 form-group">
				<label class="control-label" for="fakeFileInput">파일 선택</label>
				<div class="input-group">
					<input type="text" class="form-control" id="fakeFileInput" readonly>
					<span class="input-group-btn">
						<button class="btn btn-secondary" type="button" id="fakeFileButton">Select</button>
					</span>
				</div>
			</div>
		</div>

		<div class="row">
			<div class="col-xs-12 col-md-7 form-group">
				<label class="control-label" for="fakeFileInput">파일 이름</label>
				<input type="text" id="documentInput" class="form-control" name="document" value="파일:14b337334bb16fd998e380912839c2fd905f74fe56de0a7b63ea460d048ec3c6_image.png" />
			</div>
		</div>

		<textarea id="textInput" name="text" class="form-control">== 기본 정보 ==
|| 출처 ||  ||
|| 날짜 ||  ||
|| 저작자 ||  ||
|| 저작권 ||  ||
|| 기타 ||  ||
|| 설명 ||  ||

== 기타 ==
[[NamuFix]] 171016.0 버전을 이용하여 업로드된 이미지입니다.</textarea>

		<div class="form-group">
			<label class="control-label" for="licenseSelect">요약</label>
			<input type="text" id="logInput" class="form-control" name="log" />
		</div>

		<p>문서 편집을 <strong>저장</strong>하면 당신은 기여한 내용을 <strong>CC-BY-NC-SA 2.0 KR</strong>으로 배포하고 기여한 문서에 대한 하이퍼링크나 URL을 이용하여 저작자 표시를 하는 것으로 충분하다는 데 동의하는 것입니다. 이 <strong>동의는 철회할 수 없습니다.</strong></p>
	

		<div class="btns">
			<button id="uploadBtn" type="submit" class="btn btn-primary">올리기</button>
		</div>
		
		<div id="recaptcha"></div>
		<script>
			recaptchaInit('recaptcha', {
				'sitekey': '6LcUuigTAAAAALyrWQPfwtFdFWFdeUoToQyVnD8Y',
				'size': 'invisible',
				'badge': 'inline',
				'callback': function() { $("#uploadBtn").attr("disabled", true); $("#uploadForm").submit(); }
			}, function (id) {
				$("#uploadForm").attr('data-recaptcha', id);
			});
		</script>
		

	</form>

	<script>uploadInit();</script>

		</article>
	</div>

	<div class="footer-wrapper">
		<footer class="footer">
			<p>namu.wiki<span class="sep"></span><a href="mailto:support@namu.wiki">Contáctenos</a><span class="sep"></span><a href="/Policy">Términos de uso</a><span class="sep"></span>Operado por umanle S.R.L.<span class="sep"></span>Hecho con &lt;3 en Asunción, República del Paraguay</p>
			<p>Su zona horaria es <span id="timezone"></span><span class="sep"></span>Impulsado por <i>the seed engine</i></p>
		</footer>
	</div>

	<ul class="nav-control">
		<li>
			<a href="#top" id="goTopBtn">
				<span class="ion-arrow-up-b"></span>
			</a>
		</li>
		<li>
			<a href="#bottom" id="goBottomBtn">
				<span class="ion-arrow-down-b"></span>
			</a>
		</li>
	</ul>

	<div class="modal fade" id="shortcutHelpModal">
		<div class="modal-dialog">
			<div class="modal-content">
				<div class="modal-header">
					<button type="button" class="close" data-dismiss="modal" aria-label="Close">
					<span aria-hidden="true">&times;</span>
					<span class="sr-only">Close</span>
					</button>
					<h5 class="modal-title">단축키 도움말</h5>
				</div>
				<div class="modal-body">
					<dl class="dl-horizontal">
						<dt class="col-sm-3"><b>F</b>ront Gate</dt>
						<dd class="col-sm-9">대문</dd>

						<dt class="col-sm-3">Recent<b>C</b>hanges</dt>
						<dd class="col-sm-9">최근 변경</dd>

						<dt class="col-sm-3">Recent<b>D</b>iscuss</dt>
						<dd class="col-sm-9">최근 토론</dd>

						<dt class="col-sm-3">R<b>A</b>ndom</dt>
						<dd class="col-sm-9">랜덤 문서</dd>

						<dt class="col-sm-3"><b>Shift</b> + <b>S</b>earch</dt>
						<dd class="col-sm-9">검색</dd>

						<dt class="col-sm-3"><b>E</b>dit</dt>
						<dd class="col-sm-9">문서 편집</dd>

						<dt class="col-sm-3"><b>?</b></dt>
						<dd class="col-sm-9">단축키 도움말</dd>
					</dl>
				</div>
			</div>
		</div>
	</div>

	<div class="modal fade" id="searchModal" data-backdrop="false">
		<div class="modal-dialog modal-sm">
			<div class="modal-content">
				<div class="modal-header">
					<button type="button" class="close" data-dismiss="modal" aria-label="Close">
					<span aria-hidden="true">&times;</span>
					<span class="sr-only">Close</span>
					</button>
					<h6 class="modal-title">검색</h6>
				</div>
				<div class="modal-body">
					<div class="input-group search-box">
						<input type="text" id="searchInput" class="form-control" placeholder="Search" tabindex="1">
						<span class="input-group-btn">
							<button id="searchBtn" class="btn btn-secondary" type="button" style="margin-left: 0;">
								<span class="icon ion-search"></span>
							</button>
							<button id="goBtn" class="btn btn-secondary" type="button">
								<span class="icon ion-arrow-right-c"></span>
							</button>
						</span>
					</div>
				</div>
			</div>
		</div>
	</div>

	<div class="modal" id="footnoteModal">
		<div class="modal-dialog" role="document">
			<div class="modal-content">
				<div class="modal-header">
					<h5 class="modal-title">각주:</h5>
				</div>
				<div class="modal-body"></div>
				<div class="modal-footer">
					<button type="button" class="btn btn-block" data-dismiss="modal">닫기</button>
				</div>
			</div>
		</div>
	</div>

</body>
</html>

**Comment by LiteHell** [comment]
reCAPTCHA 오류네요. 빠른 시일내로 해결하겠습니다.

**Comment by LiteHell** [comment]
171021.1 버전에서 고쳐봤는데, 한번 업데이트해서 시도해주실 수 있으신가요?

**Comment by LiteHell** [comment]
일단 이슈 닫겠습니다. 아직도 이 현상이 발견되면 다시 이슈 열어주세요.
